docs/test_plan/exam_configuration.rst

```diff
@@ -0,0 +1,189 @@
+Exam Setup and Configuration
+============================
+
+Exam Configuration (edx-exams)
+------------------------------
+
+Prerequisite Steps:
+   #. Make sure `course_apps.exams_ida` CourseWaffleFlag is enabled for your test course
+   #. An LTI proctoring provider should be chosen for the course
```

Comment:
we're presumably using a fake one? can we put setup instructions in a section here, or maybe in another doc?

Reply:
humm yeah maybe I can lean on the read the docs pages more here and elsewhere in this doc

Reply:
Agree. As this is written, I won't know how to do this